label_studio/ml/models.py

```diff
@@ -391,6 +529,16 @@ def _validate_ml_api_result(ml_api_result, tasks, curr_logger):
     return True
 
 
+@receiver(pre_delete, sender=MLBackend)
+def modify_project_model_version(sender, instance, **kwargs):
+    """ """
```

Comment:
We generally discourage the use of Django signals because they aren't called under various conditions. why is this necessary? A docstring would really help.